pages/styles.css

```diff
@@ -0,0 +1,337 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+/* All Pages */
+body {
+    font-family: 'Montserrat';
+    margin: 0;
+}
+
+html, 
+nav {
+    background-color: #2DC7FF;
+}
+
+footer {
+    padding: .1em;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    width: 100%;
+    bottom: 0px;
+}
+
+footer,
+#navbar,
+#nav-background-home {
+    background-color: #0081AF;
+    color: #EAD2AC;
+}
+
+header {
+    padding-top: 10vh;
+    padding-left: 1em;
+}
+
+/* Navbar */
+
+#navbar {
+    display: flex;
+    justify-content: flex-end;
+    list-style-type: none;
+    position: fixed;
+    width: 95%;
+    border-radius: 0 25px 25px 0;
+    margin: 0;
+}
+
+#nav-background {
+    background: #2DC7FF;
+    padding-top: 8px;
+    position: fixed;
+    width: 100%;
+    height: 50px;
+}
+
+#navbar a {
+    color: #EAD2AC;
+    text-decoration: none;
+}
+
+#navbar a:hover {
+    color: #2DC7FF;
+}
```

Comment:
Consider using [CSS variables](https://stackoverflow.com/questions/1875852/how-can-i-define-colors-as-variables-in-css) for colors you re-use in different places.